RubeesEat/wwwroot/css/site.css

```diff
@@ -0,0 +1,172 @@
+/*source: https://www.joshwcomeau.com/css/custom-css-reset/*/
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
+    text-align: center;
+}
+
+body {
+    line-height: 1.5;
+    -webkit-font-smoothing: antialiased;
+}
+
+img, picture, video, canvas, svg {
+    display: block;
+    max-width: 100%;
+}
+
+input, button, textarea, select {
+    font: inherit;
+}
+
+p, h1, h2, h3, h4, h5, h6 {
+    overflow-wrap: break-word;
+    text-align: center;
+}
+
+#root, #__next {
+    isolation: isolate;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #2d332f;
+    color: white;
+}
+
+nav ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    align-items: center;
+}
+
+nav ul li {
+    margin-right: 1.5em;
+    transition: all 0.3s ease;
+}
+
+nav ul li a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+nav ul li:hover {
+    transform: scale(1.15);
+    z-index: 1;
+}
+
+.logo {
+    width: 50px;
+    height: 50px;
+    margin-left: auto;
+}
+
+nav {
+    background-color: #fbbf00;
+    padding: 1em 2em;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.nav-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.logo-container {
+    display: flex;
+    align-items: center;
+}
+
+.logo {
+    width: 50px;
+    height: 50px;
+}
+
+.nav-menu {
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+.nav-menu li {
+    margin-right: 1.5em;
+}
+
+.nav-menu li a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.hamburger {
+    display: none;
+    background: none;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+}
+
+.hamburger span {
+    display: block;
+    width: 25px;
+    height: 3px;
+    background-color: black;
+    margin: 5px 0;
+    transition: all 0.3s ease;
+}
+
+@media (max-width: 512px) {
```

Comment:
Please but all the break points into CSS variables and use them instead. This allows us to change the breakpoints very easily.

Reply:
Also CSS is usually developed mobile first. So all the main rules should apply to mobile. If the screen gets bigger we change stuff with media rules. So we would assert `min-width: xxx` instead and rewrite those rules.

Reply:
I tried using CSS variables, but it didn't work for media queries. When I run it through a CSS validator, I get a parse error. Apparently it's not supported to use CSS variables in media queries: https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries

I will put the media queries at the top, after root, to be easily changeable

Reply:
Okay well, then we can leave them as is. I wouldn't move them too much as they should be close to the non media rules they semantically belong to.